plugins/talk-plugin-permalink/client/components/PermalinkButton.js

```diff
@@ -36,7 +36,9 @@ export default class PermalinkButton extends React.Component {
   };
 
   copyPermalink = () => {
+    this.permalinkInput.focus();
     this.permalinkInput.select();
+    this.permalinkInput.createTextRange();
```

Comment:
It seems that this is not a valid method for input elements: https://developer.mozilla.org/en-US/docs/Web/API/HTMLInputElement

Reply:
So you mean before my PR it was fine ? 
it suppose an open task !